hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -649,9 +649,8 @@
                This is a shortcut mechanism used by the software to completely
                kill flash in case of emergency.
 
-               Since this register is rw0c instead of rw, to disable, write any value in the form of
-               0xxx or xxx0, where x could be either 0 or 1.
-
+               Since this register is rw1s instead of rw, to disable, write the value kMuBi4True
+               to the register to disable the flash.
```

Comment:
Isn't flash supposed to enable only if the pattern is precisely correct? I thought we wanted any glitching or other funny business here to disable flash with high probability.

So it seems like it should be to write `kMuBi4False` to _enable_, with any other value disabling it.

Reply:
Yeah so the RTL implements it that way:
https://github.com/lowRISC/opentitan/blob/ca68f8f9d617488b800d1a37bf3206156f150242/hw/ip/flash_ctrl/rtl/flash_ctrl_arb.sv#L155-L159

i.e., it checks for `true_loose`. 

Since this is W1S, and the reset value is `False`,  writing `True` to it will maximize the Hamming distance. 
But other non-true values would work as well.

Reply:
Ah, I see, I was confused by the description. Looks like this is a software binary-compatible change.